src/node_metadata.cc

```diff
@@ -99,6 +103,21 @@ Metadata::Versions::Versions() {
   openssl = GetOpenSSLVersion();
 #endif
 
+#ifdef __GLIBC__
+  glibcCompiler = __GLIBC__ + "." + __GLIBC_MINOR__;
```

Comment:
This line does not what you think it does (it adds offset + pointer + offset, giving you a `const char*` to random data bytes)

```suggestion
  glibcCompiler =
      NODE_STRINGIFY(__GLIBC__)
      "."
      NODE_STRINGIFY(__GLIBC_MINOR__);
```

(compare e.g. the `llhttp` line above)

Reply:
Thanks! Fixed in 3f55827